internal/cluster/transition.go

```diff
@@ -409,14 +410,27 @@ func (th *transitionHandler) enoughMastersAndWorkers(sCluster *stateCluster, sta
 	return false
 }
 
+//check if installation reach to timeout
+func (th *transitionHandler) IsInstallationTimedOut(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) (bool, error) {
+	sCluster, ok := sw.(*stateCluster)
+	if !ok {
+		return false, errors.New("IsInstallationTimedOut incompatible type of StateSwitch")
+	}
+	// can happen if the service was rebooted or somehow the async part crashed.
+	if time.Since(time.Time(sCluster.cluster.InstallStartedAt)) > th.installationTimeout {
```

Comment:
seems like ``InstallStartedAt`` is already ``time.Time``
can we remove the ``time.Time(...)``?

Reply:
it's `strfmt.DateTime`

Reply:
I think it's a reference to ``time.Time``
Anyway, not critical